<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerProfilePageCls</class>
 <widget class="QMainWindow" name="CustomerProfilePageCls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonAddCustomer">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MÜŞTERİ EKLE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEditCustomer">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MÜŞTERİ DÜZENLE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReturn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GERİ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCustomerName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>MÜŞTERİ ADI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>BARKOD ÖN EKİ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditBarcodePrefix">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSowPrefix">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>İŞ EMRİ ÖN EKİ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>SOIR SÜTUNU</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>MÜŞTERİ TARİH TİPİ</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDateTimeType">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditCustomerName</tabstop>
  <tabstop>lineEditBarcodePrefix</tabstop>
  <tabstop>lineEditSowPrefix</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>comboBoxDateTimeType</tabstop>
  <tabstop>pushButtonAddCustomer</tabstop>
  <tabstop>pushButtonEditCustomer</tabstop>
  <tabstop>pushButtonReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
